developer-docs/README.md

```diff
@@ -73,6 +73,10 @@ We cut releases from the `main` branch, following these steps:
 
 - Bump the version in `wheel_version.bzl`, `sematic/versions.py`,
   and `helm/sematic-server/Chart.yaml`
+- Increment the minor version of the `version` field in
```

Comment:
it was easy to push the PR without updating the minor version of `Chart.yaml`, given that before now that happened much lower in the procedure (after the PR was likely already merged)